packages/backend/src/managers/playgroundV2Manager.ts

```diff
@@ -216,13 +251,17 @@ export class PlaygroundV2Manager implements Disposable {
    * @param stream
    */
   private async processStream(conversationId: string, stream: Stream<ChatCompletionChunk>): Promise<void> {
+    const conversation = this.#conversationRegistry.get(conversationId);
+    if (conversation === undefined) throw new Error(`conversation with id ${conversationId} not found.`);
```

Comment:
This error is not being catched by the telemetry would be a good plus

Reply:
I took the liberty to improve the code by moving the logic of always checking `if undefined...` to the conversation registry, in the end simplifying both the PlaygroundManager and the ConversationRegistry